gst-plugin/src/drpai-models/drpai-ssdv3/ssdv3_post_processor.h

```diff
@@ -0,0 +1,45 @@
+//
+// Created by kiefer on 08/21/25.
+//
+
+#ifndef GSTREAMER1_0_DRPAI_SSDv3_POST_PROCESSOR_H
+#define GSTREAMER1_0_DRPAI_SSDv3_POST_PROCESSOR_H
+
+#include "../base_post_processor.h"
+#include <vector>
+#include <string> 
+#include <cstdint>
+
+class SSDV3_PostProcessor final: public BasePostProcessor {
+
+public:
+    explicit SSDV3_PostProcessor(const std::string& prefix);
+    ~SSDV3_PostProcessor() override = default;
+
+    void extract_detections(const std::vector<float>& inference_output_buf) override;
+
+    // load-once lifecycle hook
+    void open_resource(
+        uint32_t inference_output_size,
+        uint32_t img_width,
+        uint32_t img_height,
+        uint32_t num_classes
+    ) override;
+
+private:
+    void softmax(std::vector<float>& val);
+
+    // priors storage: [N*4] in cx, cy, w, h (norm space)
+    std::vector<float> priors_flat_;
+    uint32_t priors_count_ = 0;
```

Comment:
Remove the count. `std::vector` increases and keeps the count inside it as you call the `push_back` function.